site/_layouts/docs.html

```diff
@@ -2,176 +2,176 @@
 layout: page_submenu
 
 sidebar:
-- text: Overview
```

Comment:
Not a big deal, but these changes are curious. As far as I can tell, the only thing that changed is the tab spacing. But I'm not sure why/what caused it?

Reply:
I set my text editor to auto format with prettier. I made one change

```
  - text: Value
    url: selection-value
```

became just

```
  - text: Value
    url: value
```

sorry for the noise in the pull request.